supervision/keypoint/annotate.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional, Tuple
+
+import cv2
+import numpy as np
+
+from supervision.annotators.base import ImageType
+from supervision.draw.color import Color
+from supervision.keypoint.core import KeyPoints
+from supervision.keypoint.skeletons import resolve_skeleton_by_vertex_count
+from supervision.utils.conversion import convert_for_annotation_method
+
+
+class BaseKeyPointAnnotator(ABC):
+    @abstractmethod
+    def annotate(self, scene: ImageType, detections: KeyPoints) -> ImageType:
+        pass
+
+
+class VertexAnnotator(BaseKeyPointAnnotator):
```

Comment:
I renamed the annotator to `VertexAnnotator`. In general, I'd like to use the vertex and edge naming conventions.

Reply:
Good. At the back of our minds, let's also start thinking of names for triplets (joints?)

Having those will let us compute angles, compare skeletons and create new annotators.

Reply:
Should we change `SkeletonAnnotator` to `EdgeAnnotator` too?